docs/se/grpc/server.adoc

```diff
@@ -619,7 +612,7 @@ Typically the units value is one of the constants from `org.eclipse.microprofile
 
 ==== Overriding the Metric Name
 
-By default the metric name is the gRPC service name followed by a dot ('.') followed by the method name.
+By default, the metric name is the gRPC service name followed by a dot ('.') followed by the method name.
```

Comment:
546: "standard Helidon Metric Registries" -> "standard Helidon metric registries
546-547: "such as vendor and
application registry," -> "such as the vendor and application registries"
547: "via standard" -> "via the standard"
563: "Meta-data" -> "Metadata"
565 and 566: "meta-data" -> "metadata"
583, 597, 610, 643: The initial "the" in the callouts should be capitalized.
610: "Metric" -> "metric"
611: "from xxx class" -> "from the xxx class" (add "the")
634: "lambda" -> "a lambda expression"